### ISSUE TITLE
Phoenix Wright Ace Attorney 1

### ISSUE THREAD

**Issue by AndresVScse** [rafaelvcaetano/melonDS-android#73]
* **Device model:** SM-A107M
* **Android version:** Android API 29

**Problem:**  
Cant run the game.

**Stack trace:**  
```
java.lang.IllegalStateException: java.io.FileNotFoundException: No item at content://media/external/downloads/83213
	at android.os.Parcel.createException(Parcel.java:2096)
	at android.os.Parcel.readException(Parcel.java:2056)
	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:188)
	at android.database.DatabaseUtils.readExceptionWithFileNotFoundExceptionFromParcel(DatabaseUtils.java:151)
	at android.content.ContentProviderProxy.openTypedAssetFile(ContentProviderNative.java:705)
	at android.content.ContentResolver.openTypedAssetFileDescriptor(ContentResolver.java:1694)
	at android.content.ContentResolver.openAssetFileDescriptor(ContentResolver.java:1510)
	at android.content.ContentResolver.openInputStream(ContentResolver.java:1194)
	at me.magnum.melonds.q.i.e(Unknown Source:9)
	at me.magnum.melonds.ui.emulator.j$e.a(Unknown Source:17)
	at f.a.b0.e.b.c.m(Unknown Source:10)
	at f.a.h.a(Unknown Source:14)
	at f.a.b0.e.b.k$b.run(Unknown Source:4)
	at f.a.x.b.b$b.run(Unknown Source:2)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:237)
	at android.app.ActivityThread.main(ActivityThread.java:8019)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1100)

```

**Comment by shabbirmahmud** [comment]
Can you use the latest beta [v1.3.0](https://github.com/rafaelvcaetano/melonDS-android/releases/tag/beta-1.3.0) and open a new issue? The latest version does not fix the problem but includes additional info that will help to find the cause.

**Comment by rafaelvcaetano** [comment]
If your problem is not yet fixed, try the latest [Beta 1.4.0](https://github.com/rafaelvcaetano/melonDS-android/releases/latest) release. If the problem is not fixed, please open a new issue.
